CONTRIBUTING.md

```diff
@@ -768,9 +779,10 @@ If someone left an approving review on your PR and didn't merge a few days later
 
 Please see it as your responsibility to actively remind reviewers of your open PRs.
 
-The easiest way to do so is to simply cause them a Github notification.
-Github notifies people involved in the PR when you add a comment to your PR, push your PR or re-request their review.
+The easiest way to do so is to cause them a Github notification.
+Github notifies people involved in the PR whenever you add a comment to your PR, push your PR or re-request their review.
 Doing any of that will get you people's attention again.
+Everyone deserves proper attention, and yes that includes you!
```

Comment:
Hm, not sure how I feel about this piece of advice. I really do want to get across that committer attention isn't infinite or can in any way be *expected*.

Reply:
I myself struggled for a long time to overcome the feeling that I don't deserve to take up any space. Asking for attention is a major blocker for some.

Reply:
this is also why i prefer to avoid terms like "simply", since it is not that simple for many

Reply:
I can certainly feel where you're coming from here. Perhaps there's a wording that captures both realities.

Reply:
```suggestion
Everyone deserves proper attention, and yes that includes you! Please be mindful that committers are not always able to give everyone the attention they deserve however.
```

Reply:
:shipit: